test/integration/certificates/issuing_controller_test.go

```diff
@@ -47,12 +47,12 @@ import (
 // certificate, ca, and private key is stored into the target Secret to
 // complete Issuing the Certificate.
 func TestIssuingController(t *testing.T) {
-	config, stopFn := framework.RunControlPlane(t)
-	defer stopFn()
-
-	ctx, cancel := context.WithTimeout(context.Background(), time.Second*20)
+	ctx, cancel := context.WithTimeout(context.Background(), time.Second*40)
```

Comment:
Why has this timeout doubled?

Reply:
This was the timeout that I found to work for most tests (should not give context timeout errors).
(I just set all timeouts to the same value)